nitrous-post-create.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+rm -rf ~/code/example
+
+export LANG=en_US.UTF-8
+export LANGUAGE=en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+export DEBIAN_FRONTEND=noninteractive
+
+sudo apt-get update
+sudo -E apt-get -q -y install mysql-server
+sudo apt-get -y install libmysqlclient-dev
+sudo apt-get clean
+
+npm install -g phantomjs
+cp .env.example .env
```

Comment:
I've added `pwgen` to generate `DATABASE_PASSWORD` and `APP_SECRET_TOKEN` in `.env`  and also use `huginn` as `DATABASE_USERNAME`.